server.go

```diff
@@ -531,7 +531,6 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 
 	//nolint:lll
 	featureMgr, err := feature.NewManager(feature.Config{
-		NoStaticRemoteKey:        cfg.ProtocolOptions.NoStaticRemoteKey(),
```

Comment:
Same here. I'm not sure it makes sense to keep the command line flag but make it non-functional.